.github/workflows/realm-generator.yml

```diff
@@ -0,0 +1,64 @@
+name: Realm Generator
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'generator/**'
+      - .github/workflows/realm-generator.yml
+
+  pull_request:
+    paths:
+      - 'generator/**'
+      - .github/workflows/realm-generator.yml
+
+jobs:
+  Generator_CI:
```

Comment:
we use `CI` name  for the others. should this be any different?

Reply:
I wonder why do we name it differently? Any value in that?

Reply:
It shows up separately in the github UI. 

![image](https://user-images.githubusercontent.com/22237677/152848068-8e0f6312-5c04-497a-ac6f-86f228fecae0.png)

Reply:
Notice btw the failed steps, that don't break the job